frontend/simple/js/state.js

```diff
@@ -7,17 +7,21 @@ import Vue from 'vue'
 import Vuex from 'vuex'
 import EventLog from './event-log'
 import {EVENT_TYPE} from '../../../shared/constants'
-import userSession from './user-session'
+import {makeUserSession} from '../../../shared/functions'
+import localforage from 'localforage'
+import _ from 'lodash-es'
 const {SUCCESS, ERROR} = EVENT_TYPE
 
 Vue.use(Vuex)
-
+let _userSessions = localforage.createInstance({
+  name: 'Group Income',
+  storeName: 'User Sessions'
+})
 // TODO: save only relevant state to localforage
 const state = {
-  currentLogPosition: null,
+  session: null,
```

Comment:
Strongly recommend adding a comment here to explain what `session` is (i.e. it's the data that's saved between app relaunches).